aten/src/ATen/core/boxing/KernelFunction_impl.h

```diff
@@ -34,10 +34,16 @@ inline bool KernelFunction::isFallthrough() const {
 inline void KernelFunction::callBoxed(const OperatorHandle& opHandle, Stack* stack) const {
     if (C10_UNLIKELY(boxed_kernel_func_ == nullptr)) {
         if (unboxed_kernel_func_ == nullptr) {
-            TORCH_INTERNAL_ASSERT(false, "Tried to call KernelFunction::callBoxed() on an uninitialized KernelFunction.");
+            TORCH_INTERNAL_ASSERT(
+                false,
+                "Tried to call KernelFunction::callBoxed() on an uninitialized KernelFunction. ",
+                dumpOperatorHandle(opHandle));
```

Comment:
OperatorHandle is only forward declared here - Dispatcher.h includes this header, so creating a dump method to break the circular dep.

Reply:
Here's an alternative which is better for binary size: move the error reporting functionality into the cpp file (so lines 36 to 48 as seen here. Then you don't have to deal with the forward declaration, and we don't repeatedly inline the error reporting code here.

Reply:
Updated - also added some mobile selective build specific info as this is where it frequently happens - hope you don't feel it's too verbose. cc: @smessmer